.gitignore

```diff
@@ -2,3 +2,5 @@
 *~
 .env/*
 django_multitenant.egg-info/
+.venv
```

Comment:
nit:, isn't `.venv` a directory aswell? Other directories have a `/` suffix.

Reply:
nope, for me it's file with the name of the environment (all environments are then stored somewhere in `~/.virtualenvs` or something